### ISSUE TITLE
Could not find a declaration file for module 'seedrandom'.

### ISSUE THREAD

**Issue by t3chguy** [Callidon/bloom-filters#75]
New version of https://github.com/Callidon/bloom-filters/issues/72

The package makes references to `seedrandom` types in the compiled artifact without it being a dependency (its only a devDep).

![image](https://github.com/user-attachments/assets/529faa04-98b9-4bcf-8c14-4c8731f90006)

```
$ tsc --noEmit --jsx react && tsc --noEmit --jsx react -p playwright
node_modules/bloom-filters/dist/base-filter.d.ts:1:34 - error TS7016: Could not find a declaration file for module 'seedrandom'. '/Users/t3chguy/WebstormProjects/element-web/node_modules/seedrandom/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/seedrandom` if it exists or add a new declaration (.d.ts) file containing `declare module 'seedrandom';`

1 import seedrandom, { PRNG } from 'seedrandom';

**Comment by folkvir** [comment]
The fix has been released in the v3.0.3. If you still have any trouble don't hesitate to comment again we will reopen the issue.

**Comment by t3chguy** [comment]
Thanks @folkvir - can confirm `v3.0.3` fixes the issue
